client/src/pages/public/ParticipantLogin.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { Page, Button } from '../../components/generic';
+import { useHistory } from 'react-router-dom';
+import { useKeycloak } from '@react-keycloak/web';
+import Box from '@material-ui/core/Box';
+import { makeStyles } from '@material-ui/core/styles';
+import { Routes } from '../../constants';
+import store from 'store';
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    color: 'white',
+    marginTop: '10px',
+  },
+  blueText: {
+    color: theme.palette.primary.light,
+  },
+  blueBox: {
+    backgroundColor: '#EDF6FF',
+    width: 554,
+    height: 205,
+  },
+  rightContainer: {
+    backgroundColor: '#FFFFFF',
+    height: 'calc(100vh - 68px)',
+  },
+  leftContainer: {
+    height: 'calc(100vh - 68px)',
+  },
+}));
+
+export default () => {
+  const classes = useStyles();
+  const [keycloak] = useKeycloak();
+  const history = useHistory();
+  const redirectToLogin = () => {
+    store.set('REDIRECT', true);
```

Comment:
How does setting REDIRECT work here? I would expect it to hold a url rather than a boolean value, but I've also never used it before.

Reply:
I copied this over from Login.js, but it looks like it doesn't do anything here. I will remove it.